.spin/cmds.py

```diff
@@ -465,29 +419,6 @@ def ipython(ctx, ipython_args):
     util.run(["ipython", "--ignore-cwd",
               f"--TerminalIPythonApp.exec_lines={preimport}"] +
              list(ipython_args))
-
-
-@click.command(context_settings={"ignore_unknown_options": True})
-@click.argument("args", nargs=-1)
-@click.pass_context
-def run(ctx, args):
-    """🏁 Run a shell command with PYTHONPATH set
-
-    \b
-    spin run make
-    spin run 'echo $PYTHONPATH'
-    spin run python -c 'import sys; del sys.path[0]; import mypkg'
-
-    If you'd like to expand shell variables, like `$PYTHONPATH` in the example
-    above, you need to provide a single, quoted command to `run`:
-
-    spin run 'echo $SHELL && echo $PWD'
-
-    On Windows, all shell commands are run via Bash.
-    Install Git for Windows if you don't have Bash already.
-    """
-    ctx.invoke(build)
-    ctx.forward(meson.run)
```

Comment:
The [upstream docstring](https://github.com/scientific-python/spin/blob/b5ea9af686c20db254ae3f278d3defc626add2c1/spin/cmds/util.py#L9) does not mention the `PYTHONPATH`. Is that intentional?

Reply:
A bit confusing, but the `click` command upstream equivalent is here and mentions it: https://github.com/scientific-python/spin/blob/b5ea9af686c20db254ae3f278d3defc626add2c1/spin/cmds/meson.py#L462